src/pages/index.js

```diff
@@ -24,7 +24,7 @@ const IndexPage = ({data}) => (
             <hr className="horizontal-rule" />
                 <div className="columns is-multiline is-centered">
                   {data.allStrapiProject.edges.map(document => (
-                      <div className="column is-8-tablet is-6-desktop is-3-widescreen">
+                      <div key={document.node.id} className="column is-8-tablet is-6-desktop is-3-widescreen">
```

Comment:
I get that keys should be included in lest elements, just still not 100% sure why even after looking it up.

Reply:
I'm not sure either, but I know that I see console errors if it's not there...